ui/app/i18n/clinical/locale_en.json

```diff
@@ -197,6 +197,7 @@
   "MESSAGE_DIALOG_OPTION_COPY": "Copy Error",
   "MESSAGE_DIALOG_OPTION_OKAY": "OK",
   "LAB_ENTRY_KEY": "Lab Entry",
+  "ESANJEEVANI_KEY": "e-Sanjeevani",
```

Comment:
This code is being put in Bahmni Product codebase, and not in clinic-config? I has assumed that you can add a nav link via Configuration itself. https://bahmni.atlassian.net/wiki/spaces/BAH/pages/114219028/Navigation+Links+Control

Reply:
Yeah @gsluthra I updated the PR accordingly now